server/fly-away/src/main/java/com/codestates/flyaway/domain/board/service/BoardService.java

```diff
@@ -0,0 +1,71 @@
+package com.codestates.flyaway.domain.board.service;
+
+import com.codestates.flyaway.domain.board.entity.Board;
+import com.codestates.flyaway.domain.board.repository.BoardRepository;
+import com.codestates.flyaway.global.dto.MultiResponseDto;
+import com.codestates.flyaway.global.exception.BusinessLogicException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import static com.codestates.flyaway.global.exception.ExceptionCode.*;
+import static com.codestates.flyaway.web.board.dto.BoardDto.*;
+import static com.codestates.flyaway.web.board.dto.BoardDto.BoardResponseDto.*;
+
+@Transactional(readOnly = true)
+@Service
+@RequiredArgsConstructor
+public class BoardService {
+
+    private final BoardRepository boardRepository;
+
+    @Transactional
+    public BoardResponseDto create(Create createDto) {
```

Comment:
```suggestion
    public BoardResponseDto create(BoardDto.Create createDto) {
```

inner class 활용은 팀의 선택입니다.
하지만 활용하기로 했다면 type 이 dto 임을 명시적으로 알려주세요.
변수명만으로는 헷갈립니다.